release/release_tests.yaml

```diff
@@ -10,7 +10,7 @@
 #  working_dir: example_dir
 #
 #  # How often to run the tests.
-#  # One of [disabled, any, multi, nightly, weekly].
+#  # One of [manual, any, multi, nightly, weekly].
```

Comment:
I think I'm still a bit confused about the difference between `manual` and `any`.